fmdpy/download.py

```diff
@@ -1,14 +1,25 @@
 """Downloader for fmdpy."""
 import os
-#import sys
+import subprocess
 import tempfile
-import requests
-import ffmpeg
-import music_tag
+
 import lyricsgenius
+import music_tag
+import requests
 from tqdm import tqdm
-from fmdpy import headers, config, utils
 
+from fmdpy import config, headers, utils
+
+
+def convert_audio_to_mp3(input_file_path, output_file_path, bitrate):
```

Comment:
This will fail for '.opus' output file names. (Probably because of the codec option!)

Reply:
Converted to a generic function.